src/System Application/App/Extension Management/src/ExtensionManagement.Page.al

```diff
@@ -182,6 +183,19 @@ page 2500 "Extension Management"
                         ExtensionOperationImpl.DownloadExtensionSource(Rec."Package ID");
                     end;
                 }
+                action("Open Source in VS Code")
+                {
+                    Caption = 'Open Source in VS Code';
+                    Enabled = IsSourceSpecificationAvailable;
+                    Image = Download;
+                    Scope = Repeater;
+                    ToolTip = 'Open the source code for the extension based on the source control information.';
+
+                    trigger OnAction()
+                    begin
+                        Hyperlink(VsCodeRequestManagement.GetUrlToOpenExtensionSource(Rec."Source Repository Url", Rec."Source Commit ID"));
```

Comment:
I'd make this so the module has a method which will also do the Hyperlink. When i use the module I'd just want to call something like:
SourceCodeIntegration.ViewSourceCode() 
(maybe with a param or two :P or just pass a Rec? )

and then I wouldn't need to know the 'how' of how to open... could be VS Code, could be a GitHub url...